### PR TITLE
Add GET /object-storage/transfer/

### PR DESCRIPTION
**Pull request by leslitagordita**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -11850,6 +11850,46 @@ paths:
...
+        Object Storage adds 1 terabyte of outbound data transfer to your data transfer pool.
+        See the [Object Storage Pricing and Limitations](/docs/guides/pricing-and-limitations/)
+        guide for details on Object Storage transfer quotas.
+
+      tags:
+      - Object Storage
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - object_storage:read_write
```
Comment (adammohammed) [inline openapi.yaml:11871]: This should be `object_storage:read_only`

File: openapi.yaml [hunk 2]
```diff
@@ -11850,6 +11850,46 @@ paths:
...
+      responses:
+        '200':
+          description: A paginated list of buckets you own.
```
Comment (adammohammed) [inline openapi.yaml:11874]: Does this need to have a description? The transfer description below should suffice. This endpoint is also not paginated.

**Review by adammohammed — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (leslitagordita) [reply openapi.yaml]: Updated!

File: openapi.yaml (hunk 2 above)
Comment (leslitagordita) [reply openapi.yaml]: Our theme requires a description here. I've updated the description copy.

**Review by adammohammed — APPROVED** [review]
Looks good, thank you!

**Pull request merged**